_posts/2023-03-13-introducing-security-analytics.md

```diff
@@ -0,0 +1,72 @@
+
+(Dev or Jimish - provide screen grab of the Overview page lit up like a Christmas tree. Or, one of the dashboards packed full of graphics.)
+
+By integrating these separate components into one operation, we arrive at a solution that can detect and identify suspicious events in the network, alert security operations to the possibility of an attack, and display findings in real time so that users can inspect and evaluate events to determine whether they need to take action.
+
+## How it works
+
+Security Analytics interprets data in a company’s existing log files to identify abnormalities or suspicious patterns in network activity. After the log data is ingested in the system, detectors map the data to a common rule schema and transform it into meaningful events that can be applied to the rules. Depending on which rules have been assigned during configuration of the detector, the detector applies certain criteria to the log data and executes a match when any of the criteria are met. This generates a finding, which is simply an incident in which the detector has isolated as an event of interest. Simultaneously, the event triggers an alert, and security operations members are notified of the incident.
+
+(There could be a flow diagram graphic here to visually represent the sequence of events above)
+
+The user interface in OpenSearch Dashboards lists individual findings according to preferred metrics and displays these findings in a configurable graph so that users can drill down to see incidents in a short time frame, or expand out to get a sense of trends over a longer time period. The same format is applied to an Alerts dashboard so that users can track alerts in a selected span of time and view them by severity levels or whether or not they are still active or acknowledged.
+
+## Looking ahead
+
+Since its release in version 2.4, Security Analytics has experienced refinements to both its functionality and number of features. However, there is room for improvement and more work to do before Security Analytics becomes the comprehensive tool we envisioned at the outset of development. Here are a few of the updates we hope to address in the near future.
+
+Keeping in line with our efforts to date, we will continue to expand log-type compatibility so that the options for using log data are as extensive as the systems that generate the data. This will be an incremental effort. But the list of supported log types will increase with each new version. Some log types we plan to add in upcoming releases include Cisco ASA, Palo Alto Networks Firewall logs, Cloudfront, SecurityHub findings, and Route53, to name just a few.
```

Comment:
We should be careful in making this claim "But the list of supported log types will increase with each new version." i would revise it to what we wish to do rather than making any promises or setting the wrong expectations

Reply:
I would rephrase this as this does not install confidence in the end user that this solution is ready for production ready environment and can detect potential threats - "Since its release in version 2.4, Security Analytics has experienced refinements to both its functionality and number of features. However, there is room for improvement and more work to do before Security Analytics becomes the comprehensive tool we envisioned at the outset of development. "

Reply:
"Some log types we plan to add in upcoming releases include Cisco ASA, Palo Alto Networks Firewall logs, Cloudfront, SecurityHub findings, and Route53, to name just a few." 

while many of these are under discussion, i would not list them out here.